### PR TITLE
Jan/staging

### PR DESCRIPTION
**Pull request by jan-kiszka**
* unbreak iot2000setup
* lots of cleanups of node recipes
* update of node-red
* more powerful nc

### CODE REVIEW

**Comment by jan-kiszka** [comment]
Added two more commits, cleaning up and fixing timezone stuff.

File: meta-iot2000-example/recipes-node/node-red-contrib-boolean-logic/node-red-contrib-boolean-logic_0.0.3.bb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Recipe created by recipetool
+
+SUMMARY = "A set of Node-RED nodes for boolean logic"
+# WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
```
Comment (stormc) [inline meta-iot2000-example/recipes-node/node-red-contrib-boolean-logic/node-red-contrib-boolean-logic_0.0.3.bb:4]: It actually is MIT according to https://github.com/PerMalmberg/node-red-contrib-boolean-logic, so the comment may be removed.

**Review by stormc — APPROVED** [review]

File: meta-iot2000-example/recipes-node/node-red-contrib-boolean-logic/node-red-contrib-boolean-logic_0.0.3.bb (hunk 1 above)
Comment (jan-kiszka) [reply meta-iot2000-example/recipes-node/node-red-contrib-boolean-logic/node-red-contrib-boolean-logic_0.0.3.bb:4]: Indeed, and there are even no 10 layers of dependencies this time.
Will remove before merging.

**Comment by jan-kiszka** [comment]
merged offline

**Pull request closed without merging**